src/drive.ts

```diff
@@ -374,13 +374,13 @@ export class FileSystemDrive implements Contents.IDrive {
     let format: Contents.FileFormat;
     let fileContent: any = null;
 
-    // We assume here image, audio and video mimetypes are all and only binary files we'll encounter
-    if (
-      file.type &&
-      file.type.split('/') &&
-      ['image', 'audio', 'video'].includes(file.type.split('/')[0])
-    ) {
-      format = 'base64';
+    // We assume here text mimetypes are all and only text files we'll encounter
+    if (file.type && file.type.split('/')) {
+      if (['text'].includes(file.type.split('/')[0])) {
```

Comment:
This check seems to be treating JSON files (`application/json`) differently now.

Noticed while testing this PR on https://jupyterlab-filesystem-access--50.org.readthedocs.build/en/50/lite/lab/index.html:

![image](https://user-images.githubusercontent.com/591645/213147121-8dd0ea2f-3acc-4b63-96c7-43d3e4be43da.png)

Reply:
oops